Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "run_loop", :github => "calabash/run_loop", :branch => "develop"
-gem "json", "2.3.0"
-gem "xcpretty", ">= 0.2.8", "< 1.0"
```

Comment:
You sure that we should change fixed versions to default ones? Couldn't it potentially lead some functional to unexpected behaviour?